server/src/routes/programRoutes.ts

```diff
@@ -0,0 +1,41 @@
+import { Router, Request, Response } from 'express';
+import prisma from '../prismaClient'; // Import Prisma client
+
+const router = Router();
+
+// Route to create a new program
+router.post('/', async (req: Request, res: Response) => {
+    try {
+        const { name, description, startDate, aimAndCause } = req.body;
+
+        // Convert the date to include time (e.g., "YYYY-MM-DDT00:00:00Z")
+        const dateTime = new Date(`${startDate}T00:00:00Z`);
```

Comment:
Once again it might be that I don't have access to the database, but when I was filling out the add program form, I did not see an option to add a time to the date